source/plot/PlotData2D.ooc

```diff
@@ -0,0 +1,66 @@
+use ooc-draw
+use ooc-collections
+use ooc-math
+
+PlotData2D: abstract class {
+	label: String { get set}
+	fontSize: Int { get set }
+	lineWidth: Float { get set }
+	colorBgra: ColorBgra { get set }
+	color: String { get set }
+	opacity: Float { get set }
+	dataSerie: VectorList<FloatPoint2D> { get set }
+
+	init: func ~default() {
+		this init(VectorList<FloatPoint2D> new(), "")
+	}
+	init: func ~dataserie(dataSerie: VectorList<FloatPoint2D>) {
+		this dataSerie = dataSerie
+		this lineWidth = 1
+		this fontSize = 14
+		if (this label == null)
+			this label = ""
```

Comment:
If this is a constructor, how is label ever going to be anything but `null` before being assigned?